packages/appstore/src/components/elements/account-wizard/account-wizard.tsx

```diff
@@ -319,7 +319,7 @@ const AccountWizard: React.FC<TAccountWizard> = (props: TAccountWizard) => {
                 has_currency={has_currency}
                 form_error={form_error}
                 {...passthrough}
-                key={step_index}
+                key={step.header.title}
```

Comment:
does it need optional chaining?

Reply:
I'll add it just in case it does

Reply:
But from it's declaration, I didn't think it was optional
<img width="441" alt="def" src="https://user-images.githubusercontent.com/104334373/182781756-fd868531-a94d-45d3-9b86-68cdf6723028.png">